### ISSUE TITLE
unipept pept2 tools should not reorder and/or modify peptides

### ISSUE THREAD

**Issue by pdawyndt** [unipept/unipept#267]
Running the `unipept pept2...` tools changes the order of the peptides read from the input. In case the `-e` option is used, the tools also modify the peptides as read from the input to a normalized version. Both reordering and modification should not happen.

The modification might happen with `unipept pept2prot -e`, but then only with a specific option and the modified version of the peptide should in this case be the version of the peptide as it is found in the identified protein. This might be confusing in rare cases where multiple versions of the same peptide match the same protein though. I also don't know if the database has a fast way of identifying what version of the peptide matches a given protein in case I and L are equated.

Example:

```
$ fasta2prot < patient_TS19_CDS | sed -ne '10{p;q}' | prot2pept | peptfilter
YFGHIFSDEDK
NLLETGNMGR
VVELK
NGEYIPSFISIDK
LTNEVVAMK
AENAFIPR
GVELTEQLXTR
NEGIGK
AHEGI
$ fasta2prot < patient_TS19_CDS | sed -ne '10{p;q}' | prot2pept | peptfilter | unipept pept2lca
sequence,taxon_id,taxon_name,taxon_parent_id,taxon_rank
VVELK,1,root,1,no rank
NLLETGNMGR,976,Bacteroidetes,68336,phylum
LTNEVVAMK,171549,Bacteroidales,200643,order
$ fasta2prot < patient_TS19_CDS | sed -ne '10{p;q}' | prot2pept | peptfilter | unipept pept2lca -e
sequence,taxon_id,taxon_name,taxon_parent_id,taxon_rank
NEGLGK,1,root,1,no rank
VVELK,1,root,1,no rank
AENAFLPR,817,Bacteroides fragilis,816,species
NGEYLPSFLSLDK,817,Bacteroides fragilis,816,species
NLLETGNMGR,976,Bacteroidetes,68336,phylum
LTNEVVAMK,171549,Bacteroidales,200643,order
```

_[Original issue](https://github.ugent.be/unipept/unipept/issues/267) by @pdawyndt on Sat May 03 2014 at 10:28._
_Closed by an unknown user on Tue May 06 2014 at 15:01._

**Comment by bmesuere** [comment]
I've fixed the renaming of input peptides and also the order for a single batch.

However because multiple requests are queued there is no way to guarantee the order of batches.
I'm going to implement something that can keep track of the batch order and will output them in the correct order.


_[Original comment](https://github.ugent.be/unipept/unipept/issues/267#issuecomment-9255) by ghost on Tue May 06 2014 at 14:01._

**Comment by bmesuere** [comment]
This is fixed in unipept 0.3.1.


_[Original comment](https://github.ugent.be/unipept/unipept/issues/267#issuecomment-9258) by ghost on Tue May 06 2014 at 15:01._
